extension/pybindings/test/test_backend_pybinding.py

```diff
@@ -0,0 +1,12 @@
+import unittest
+
+from executorch.extension.pybindings.portable_lib import _get_backend_names
+
+
+class TestBackendsPybinding(unittest.TestCase):
+    def test_backend_name_list(
+        self,
+    ) -> None:
+        all_backend_name = _get_backend_names()
```

Comment:
In the unit test, use the new pybindings API

```
from executorch.runtime import Runtime

runtime = Runtime.get()

runtime.backend_registry.get_backend_names()
```

Reply:
updated